js/main.js

```diff
@@ -145,19 +147,33 @@ function viewSwap(view) {
 // Entry Form View Event Listener
 const $entryFormAnchor = document.querySelector('#new-button');
 $entryFormAnchor.addEventListener('click', event => {
+  $photoPreview.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $form.reset();
+  $newEditEntry.textContent = 'New Entry';
+  data.editing = null;
+  $deleteButton.remove();
   viewSwap('entry-form');
 });
 
 // Entries View Event Listener
 const $entriesViewAnchor = document.querySelector('#entries-button');
 $entriesViewAnchor.addEventListener('click', event => {
+  data.editing = null;
   viewSwap('entries');
 });
 
 // Edit View Event Listener
+const $deleteButton = document.createElement('a');
 const $newEditEntry = document.querySelector('#new-edit-entry');
 $ul.addEventListener('click', event => {
 
+  $deleteButton.setAttribute('href', '#');
+  $deleteButton.setAttribute('id', 'delete-button');
+  $deleteButton.textContent = 'Delete Entry';
+  const $saveButton = document.querySelector('#save-button');
+  const $deleteButtonDiv = $saveButton.closest('.column-full');
+  $deleteButtonDiv.prepend($deleteButton);
```

Comment:
This should be created in your HTML file. You should just be showing / hiding it depending if we're editing an entry or not